R/report.numeric.R

```diff
@@ -191,9 +191,9 @@ report_parameters.numeric <- function(x,
     if (isTRUE(missing_percentage)) {
       n_missing <- table$percentage_Missing[1]
       text_missing <- paste0(insight::format_value(table$percentage_Missing[1],
-        protect_integers = TRUE,
-        digits = digits
-      ), "% missing")
+                               protect_integers = TRUE,
+                               digits = digits
+                             ), "% missing")
```

Comment:
Are you doing this manually? 

Please don't do so because `{styler}` will revert it. I know that this produces `indentation_linter()` lints, but that's a bug in `{lintr}` and I have already created an issue.

So you can just ignore these lints for now.